screens/OrderFormII.js

```diff
@@ -322,6 +323,36 @@ class OrderFormII extends React.Component {
     }).then()
   }
 
+  handleFreeLineitem = (orderId, lineitemId, isFree) => {
+    const formData = new FormData()
+    formData.append('free', isFree)
+    dispatchFetchRequestWithOption(
+      api.order.freeLineItems(orderId, lineitemId),
+      {
+        method: 'PATCH',
+        withCredentials: true,
+        credentials: 'include',
+        headers: {
+          'Content-Type': 'application/json'
+        },
+        body: formData
+      }, {
+      defaultMessage: false
+    }, response => {
+      response.json().then(data => {
+        console.log('ress', data)
+        if (data.hasOwnProperty('orderId')) {
```

Comment:
I guess we don't need the if check right? If so, you can check to the following:

const message = isFree ? 'order.free' : 'order.cancelFree'
successMessage(this.context.t(message))